src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -73,7 +76,7 @@ def send_uds_raw(
             log.info(
                 f"UDS request to send '{['0x{:02X}'.format(i) for i in msg_to_send]}'"
             )
-            resp = self.uds_config.send(msg_to_send)
+            resp = self.uds_config.send(msg_to_send, tpWaitTime=tp_waiting_time)
```

Comment:
Could the tp_waiting_time eventually be added as UdsBaseAuxiliary attribute so that it can be set for the entire test and applied it the UDSServerAuxiliary also? 
https://github.com/eclipse/kiso-testing/blob/master/src/pykiso/lib/auxiliaries/udsaux/uds_server_auxiliary.py#L142

Reply:
As far I remember, If you play the rol of UDS server you only send single frame no?
If yes you never reach the else condition (see encode iso-tp from kiso-testing-python-uds in CanTP module)

Reply:
Actually, if the response to send by the server is bigger than 64 bytes, it will send the first frame and enter the wait state and the 10ms sleep could occur if no flow control is received by the client